broker/tasks/alb.py

```diff
@@ -2,6 +2,7 @@
 import time
 
 from sqlalchemy import and_
+from sqlalchemy.orm.attributes import flag_modified
```

Comment:
Interesting!  I [looked this up](https://docs.sqlalchemy.org/en/13/orm/session_api.html#sqlalchemy.orm.attributes.flag_modified) and wonder if this would've helped with the other issue we were seeing with the list value not being set, though I suspect the fix we made was still correct given Python's way of handling variable assignments.

Reply:
I think it would've worked, yeah.

I don't know which I'd call the "right" way. This way, IMO, seems more obvious as to _why_ we're doing it. The other way seems more obvious about _what_ we're doing.